src/components/Breadcrumbs/SbBreadcrumbs.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <nav class="sb-breadcrumbs" aria-label="breadcrumb" :class="activeClass">
+    <template v-if="items.length > 6">
```

Comment:
May we avoid _magical numbers_ by adding a constant to the script and passing it to the template with a meaningful name for this `6` value?